### ISSUE TITLE
Prediction graph is wrong

### ISSUE THREAD

**Issue by mbrostami** [JordiCorbilla/stock-prediction-deep-neural-learning#17]
Hi
Thanks for the work you have done.
I think the prediction graph should be shown with t+timesteps if you use test_data.index. So in this case three step further than real value.

Based on my personal experience, using LSTM to predict stock market will end up with a model that takes the most influence from the current data frame (timewindow of each step) the model is trying to predict. So you will see when price goes up model predicts in up direction and if it goes down it predicts in down direction. The main problem might come from the usual loss functions that is being used like MSE. The loss function is low when price movement and prediction are going in the same direction after some training steps. But it doesn't cover good enough the edge cases when price direction changes which means the model can not be used in real world.
